src/main/java/com/saebyeok/saebyeok/controller/MemberController.java

```diff
@@ -0,0 +1,33 @@
+package com.saebyeok.saebyeok.controller;
+
+import com.saebyeok.saebyeok.dto.LoginRequest;
+import com.saebyeok.saebyeok.dto.TokenResponse;
+import com.saebyeok.saebyeok.exception.IllegalSocialLoginException;
+import com.saebyeok.saebyeok.service.MemberService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@AllArgsConstructor
+@RequestMapping("/api")
+@RestController
+public class MemberController {
+    private final MemberService memberService;
+
+    @PostMapping("/login")
+    public ResponseEntity login(@RequestBody LoginRequest request) {
```

Comment:
반환타입을 적어주는 게 좋을 것 같아요!
```suggestion
    public ResponseEntity<TokenResponse> login(@RequestBody LoginRequest request) {
```

Reply:
추가했어요 :)